Pseudocode/Stack/Stock Span/SourceCode.tex

```diff
@@ -0,0 +1,69 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Don not remove the 'end' at the end of the algorithm
+\usepackage{algpseudocode}
+
+% Let us keep the 'end only for 'For'
+% Manually remove the 'end' for some sections
+\algtext*{EndIf}
+\algtext*{EndFunction}
+\algtext*{EndWhile}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\newcommand{\IsNotEmpty}{\textbf{ is not empty,}}
+\newcommand{\Break}{\textbf{Break}}
+\newcommand{\Print}{\textbf{Print}}
+
+\begin{document}
+
+    \begin{algorithm}
+    
+      \caption{Stock Span}
+      
+      \begin{algorithmic}[1]
+        \Ensure One Based Indexing
+        \Statex
+        \Function {$Calculate\_Stock\_Span$}{$prices,len$}
+            \LeftComment{$span[i]$ denotes the span of the price of stock at the $i-th$ day}
+            
+            \Statex
+            \For{$ i = 1 : len$}
+                \Statex
+                \While{$stack \IsNotEmpty$}
+                    \If{$price$[$stack.top$] $<=$ $price$[$i$]}
+                        \State{$stack.pop$}
+                    \Else 
+                        \State{\Break} Loop
+                    \EndIf
```

Comment:
This line is unnecessary (and it wasn't even there in the code that you shared? Why did you over complicate it?

Delete this segment